app/services/idv/steps/send_link_step.rb

```diff
@@ -28,9 +28,9 @@ def formatted_destination_phone
 
       def link(token)
         if request.path.include?('doc_auth_v2')
-          idv_doc_auth_v2_step_url(step: :scan_id, token: token)
+          idv_doc_auth_v2_step_dashes_url(step: :scan_idto_s.dasherize, token: token)
```

Comment:
Was this supposed to be `scan_id.to_s.dasherize`?

Reply:
oh yikes... yes it was